orchagent/portsorch.cpp

```diff
@@ -26,6 +26,7 @@
 #include "countercheckorch.h"
 #include "notifier.h"
 #include "redisclient.h"
+#include "fdborch.h"
```

Comment:
do you need to include this?

Reply:
yes, otherwise it is not able to get the definition of FdbOrch. I am getting compilation errors.

portsorch.cpp:44:8: error: 'FdbOrch' does not name a type; did you mean 'CrmOrch'?
 extern FdbOrch *gFdbOrch;
        ^~~~~~~
        CrmOrch
portsorch.cpp: In member function 'bool PortsOrch::removeBridgePort(swss::Port&)':
portsorch.cpp:3437:5: error: 'gFdbOrch' was not declared in this scope
     gFdbOrch->flushFDBEntries(port.m_bridge_port_id, SAI_NULL_OBJECT_ID);
     ^~~~~~~~
portsorch.cpp:3437:5: note: suggested alternative: 'gCrmOrch'
     gFdbOrch->flushFDBEntries(port.m_bridge_port_id, SAI_NULL_OBJECT_ID);
     ^~~~~~~~
     gCrmOrch